model/system_metrics_test.go

```diff
@@ -162,15 +162,17 @@ func TestSystemMetricsAppend(t *testing.T) {
 		chunk1Key, ok1 := keyCheck[string(chunk1)]
 		chunk2Key, ok2 := keyCheck[string(chunk2)]
 		assert.True(t, ok1 && ok2)
-		chunk1Nanos, err1 := strconv.ParseInt(chunk1Key, 10, 64)
-		chunk2Nanos, err2 := strconv.ParseInt(chunk2Key, 10, 64)
+		assert.Equal(t, "Test", chunk1Key[:4])
+		assert.Equal(t, "Test", chunk2Key[:4])
+		chunk1Nanos, err1 := strconv.ParseInt(chunk1Key[4:], 10, 64)
```

Comment:
Can we just use `strconv.Itoa`?

Reply:
This is going in the other direction, from string to int, and the output needs to be an int64 which is why I didn't use Atoi